testing/server.spec.js

```diff
@@ -0,0 +1,128 @@
+const request = require('supertest');
+const server = require('../server.js');
+const db = require('../data/dbConfig.js');
+const games = require('../server-model.js');
+
+describe('The Server', () => {
+
+    //server is running -> pass
+    it('checks status code:200', () => {
+        return request(server).get('/').expect(200);
+    });
+    
+    //hello message is running -> pass
+    it('checks server hello msg', () => {
+        const expected = {message: 'Hello from Patty. BE Week4 Sprint Challenge Project'};
+
+        request(server).get('/').then(res => {
+            expect(res.body).toEqual(expected)
+        });
+    });
+
+});
+
+describe('The Endpoints', () => {
+
+    //use beforeAll to run once. beforeEach to run always.
+    beforeEach(async () => {
+        await db('games').truncate();
+    });
+
+    //GET endpoint
+    it('GET/games should return code:200', () => {
+        return request(server)
+        .get('/games')
+        .then(res => {
+            expect(res.status).toBe(200);
+        })
+    });
+
+    it('GET/games should return an array', () => {
+        const expected = [];
+
+        return request(server)
+        .get('/games')
+        .then(res => {
+            expect(expected).toEqual([])
+        });
+    });
+
+    //beforeEach truncate will set this to 0.
+    it('GET/games should return an array with length 0', () => {
+        const expected = [];
+
+        return request(server)
+        .get('/games')
+        .then(res => {
+            expect(expected).toHaveLength(0)
+        });
+    });
+
+    it('GET/games should return Content-Type application or json', () => {
+        return request(server)
+        .get('/games')
+        .expect('Content-Type', /application/ || /json/)
```

Comment:
if its sending application/json this test will never go past the first check here so putting the || /json/ , this code will never run and as there are multiple types of application data that can be sent I would maybe just check for the json part, just a thought